src/main/java/com/minecolonies/coremod/colony/ColonyList.java

```diff
@@ -15,15 +15,13 @@
 import java.util.stream.StreamSupport;
 
 //TODO extend list/collection
+
 /**
  * Data structure for storing colonies, optimized for performance.
  *
- * @author Colton
- *
  * @param <T> Type of IColony (Colony or ColonyView)
  */
-public final class ColonyList<T extends IColony> implements Iterable<T>
-{
+public final class ColonyList<T extends IColony> implements Iterable<T> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)